python/nano/src/bigdl/nano/deps/openvino/core/model.py

```diff
@@ -87,17 +89,99 @@ def ie_network(self, model):
                 config = {"INFERENCE_NUM_THREADS": str(self.thread_num)}
         else:
             config = {}
-        if self.additional_config is not None and self._device == 'CPU':
-            # TODO: check addition config based on device
-            config.update(self.additional_config)
-        self._compiled_model = self._ie.compile_model(model=self.ie_network,
-                                                      device_name=self._device,
-                                                      config=config)
-        self._infer_request = self._compiled_model.create_infer_request()
+        if self.additional_config is not None:
+            if self._device == 'CPU':
+                # TODO: check addition config based on device
+                config.update(self.additional_config)
+            else:
+                ov_cache_dir = self.additional_config.get("CACHE_DIR", None)
```

Comment:
cache_dir can't be used for CPU?

Reply:
The original logic will ignore all passed config if the device == GPU.
Now CPU will update all configs, and GPU will only use the CACHE_DIR.